Driver/Driver/Helper/AppEnvironment.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  AppEnvironment.swift
+//  Driver
+//
+//  Created by Fatih Doğan on 27.02.2023.
+//
+
+import Foundation
+
+/// AppEnvironment enum reads the settings bundle and change environment according to the user's preference
+enum AppEnvironment: String {
+    case production, development
+    
+    var url: String {
+        // FIXME: Change after servis deploy
+        switch self {
+        case .development:
+            return "https://development.base.url/"
+        case .production:
+            return "https://production.base.url/"
+        }
+    }
+    
+    static var selected: AppEnvironment {
+         /**
+          Production is default value for fail safe
+          */
+             let defaultValue = AppEnvironment.production
+             if let environment = UserDefaults.standard.string(forKey: "environment") {
+               return AppEnvironment(rawValue: environment) ?? defaultValue
+             }
+             return defaultValue
```

Comment:
I think

```suggestion
        guard let environment = UserDefaults.standard.string(forKey: "environment") else {
            return .production
        }
        
        return AppEnvironment(rawValue: environment) ?? .production
```

reads better here, but just a suggestion 🙌

Reply:
Yes shorter is better I think too. I will change these lines thank you for your suggestion.